Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -304,6 +302,7 @@ - (void)updateToolbarItemEnabledState {
     self.tableOfContentsToolbarItem.enabled = self.article != nil;
     self.shareToolbarItem.enabled           = self.article != nil;
     self.languagesToolbarItem.enabled       = self.article.languagecount > 1;
+    self.tableOfContentsToolbarItem.enabled = !self.article.isMain;
```

Comment:
minor, can you combine this w/ the statement on line 302? (i.e. `enabled = self.article != nil && !self.articleisMain`). otherwise, the button will be accidentally enabled when the article is `nil`.